components/phpunit_bridge.rst

```diff
@@ -736,6 +736,7 @@ reason, this component also provides mocks for these PHP functions:
 * :phpfunction:`class_exists`
 * :phpfunction:`interface_exists`
 * :phpfunction:`trait_exists`
+* :phpfunction:`enum_exists` (requires PHP 8.1)
```

Comment:
```suggestion
* :phpfunction:`enum_exists`
```

I would remove this hint

cc @javiereguiluz WDYT?

Reply:
Yes, I think we can remove it because the entire Symfony 6.3 requires >= 8.1.

Reply:
Actually, PHP Unit Bridge requires PHP >= 7.1, which is why I put this hint. But i may remove it anyway?

Reply:
Good catch! Now I think we should keep this explicit mention to the PHP requirement 😄  Thanks

Reply:
OK, lets keep it